web/app.fluidity.money/app/root.tsx

```diff
@@ -235,15 +237,41 @@ function App() {
     }
   }, []);
 
+  const { showExperiment } = useContext(SplitContext);
+
   const [splitUser, setSplitUser] = useState(splitUserKey);
 
+  // show the user a banner if they're on our "bother"
+  // list for being interesting
+
+  const [showTargetedBanner, setShowTargetedBanner] = useState(false);
+
+  useEffect(() => {
+    if (!showExperiment("target-banner")) {
+      return;
+    }
+
+    const _targetedBannerSeen = localStorage.getItem("targetedBanner");
+    if (!_targetedBannerSeen) {
+      setShowTargetedBanner(false);
+    }
+  });
+
   return (
     <html lang="en">
       <head>
         <Meta />
         <Links />
       </head>
       <body>
+        <Banner activated={showTargetedBanner}>
+          <p>Hey there...</p>
+          <h2>Thank you for using the app!</h2>
+          <p>
+            We'd love to sign *YOU* up to our beta program, could you visit
+            <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">this link?</a>
```

Comment:
might want to change this

Reply:
No joke, hasn't been defined what it should have yet